kairon/shared/data/processor.py

```diff
@@ -5546,7 +5548,7 @@ def get_logs(self, bot: Text, logtype: str, start_time: datetime, end_time: date
                 "start_timestamp__gte": start_time,
                 "start_timestamp__lte": end_time
             }
-        value = json.loads(logs[logtype].objects(**filter_query).to_json())
+        value = json.loads(logs[logtype].objects(**filter_query).order_by('-timestamp').to_json())
```

Comment:
this is a generic method. all methods will not have timestamp field. we will need ti figure out a different way